index.js

```diff
@@ -1,296 +1,44 @@
 'use strict';
 
+var types = require('./validator/cardTypes.json');
+var clone = require('./validator/helpers/clone');
```

Comment:
I don't think it's necessary to have all these nested inside a `helpers` directory.